modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/helper/ServerPluginConfiguration.java

```diff
@@ -182,7 +182,11 @@ public Long getManagementConnectionTimeout() {
     }
 
     public Boolean isLocal() {
-        return this.pluginConfig.getSimple(Property.LOCAL).getBooleanValue();
+        Boolean isLocal = this.pluginConfig.getSimple(Property.LOCAL).getBooleanValue();
```

Comment:
This can cause NPE. ``this.pluginConfig.getSimple(Property.LOCAL)`` is equal to ``null`` as defaultValue is property is not found.

Reply:
So my recommendation here is:

```java
return Boolean.valueOf(this.pluginConfig.getSimpleValue(Property.LOCAL, "false"));
```

Reply:
Whoops, right on that

Reply:
Updated code